source/_docs/unsupported-modules-plugins.md

```diff
@@ -134,6 +134,9 @@ You are welcome to modify this patch according to your needs, such as performing
 ### Mobile Tools
 **Issue**: Conflicts with Varnish. See [https://www.drupal.org/node/1976162#comment-7411366](https://www.drupal.org/node/1976162#comment-7411366).
 <hr>
+### Node Gallery
+**Issue**: Using Node Gallery with Plupload attaches cookies to image uploads for authentication purposes. This conflicts with our Varnish configuration as we strip all cookies for images, CSS, and JS files to improve performance.
+<hr>
```

Comment:
At this time I don't know of any work-arounds for the issue. In regards to alternative modules...I don't have any ideas of something that directly replaces this functionality. @lauren-kelly do you know of any?